README.md

```diff
@@ -382,9 +386,260 @@ AnswerChallengeRequest answerChallengeRequest = AnswerChallengeRequestBuilder.bu
+credentials = new Credentials();
+
+// passcode received in email
+credentials.setPasscode("{passcode}".toCharArray());
+
+// build answer email authenticator challenge request
+AnswerChallengeRequest emailAuthenticatorAnswerChallengeRequest = AnswerChallengeRequestBuilder.builder()
+        .withStateHandle(stateHandle)
+        .withCredentials(credentials)
+        .build();
+idxResponse = remediationOption.proceed(client, emailAuthenticatorAnswerChallengeRequest);
+
+// check if we landed success on login
+if (idxResponse.isLoginSuccessful()) {
+    log.info("Login Successful!");
+    // exchange the received interaction code for a token
+    TokenResponse tokenResponse = idxResponse.getSuccessWithInteractionCode().exchangeCode(client);
+    log.info("Exchanged interaction code for token: \naccessToken: {}, \nidToken: {}, \ntokenType: {}, \nscope: {}, \nexpiresIn:{}",
+            tokenResponse.getAccessToken(),
+            tokenResponse.getIdToken(),
+            tokenResponse.getTokenType(),
+            tokenResponse.getScope(),
+            tokenResponse.getExpiresIn());
+}
+```
+[//]: # (end: loginUsingPasswordAndEmailAuthenticator)
+
+#### Login using password + phone authenticator (SMS/Voice)
+
+In this example, the Org is configured to require a Phone factor (SMS/Voice) as a second authenticator. After answering the password challenge, users have to select SMS/Voice and enter the code to finish the process.
+
+> Note: Steps to identify the user might change based on your Org configuration.
+
+[//]: # (method: loginUsingPasswordAndPhoneAuthenticator)
+```java
+// get interactionHandle
+InteractResponse interactResponse = client.interact();
+String interactHandle = interactResponse.getInteractionHandle();
+
+// exchange interactHandle for stateHandle
+IDXResponse idxResponse = client.introspect(Optional.of(interactHandle));
+String stateHandle = idxResponse.getStateHandle();
+
+// identify
+idxResponse = client.identify(IdentifyRequestBuilder.builder()
+        .withIdentifier("{identifier}") // email
+        .withStateHandle(stateHandle)
+        .build());
+
+// get remediation options to go to the next step
+RemediationOption[] remediationOptions = idxResponse.remediation().remediationOptions();
+Optional<RemediationOption> remediationOptionsOptional = Arrays.stream(remediationOptions)
+        .filter(x -> "select-authenticator-authenticate".equals(x.getName()))
+        .findFirst();
+remediationOption = remediationOptionsOptional.get();
+
+// get authenticator options
+Map<String, String> authenticatorOptions = remediationOption.getAuthenticatorOptions();
+log.info("Authenticator Options: {}", authenticatorOptions);
+
+// select phone authenticator (sms or voice)
+Authenticator phoneAuthenticator = new Authenticator();
+phoneAuthenticator.setId(authenticatorOptions.get("sms,voice"));
+
+/* id is the same for both sms and voice */
+phoneAuthenticator.setEnrollmentId(authenticatorOptions.get("enrollmentId"));
```

Comment:
enrollmentId is not required when enrolling a phone for the first time, you have to pass the phone number instead.